ios/Classes/FacebookModule.m

```diff
@@ -195,6 +195,16 @@ -(id)accessToken
     }, YES);
     return token;
 }
+
+-(id)appID
+{
+    __block NSString * appID;
```

Comment:
I just changed it here (from where you possible got it):
```objc
-(id)permissions
{
    ENSURE_UI_THREAD_0_ARGS
    return [[[FBSDKAccessToken currentAccessToken] permissions] allObjects];
}
```

Does that work? Currently have no time to build and test. It just ensures it's on the right thread (main thread) and returns the value. Should be the same for the accessToken.